scripts/fuzz_opt.py

```diff
@@ -1391,6 +1391,108 @@ def handle(self, wasm):
         compare_between_vms(output, merged_output, 'Merge')
 
 
+# Tests wasm-split
+class Split(TestCaseHandler):
+    frequency = 1  # TODO: adjust lower when we actually enable this
+
+    def handle(self, wasm):
+        # get the list of function names, some of which we will decide to split
+        # out
+        wat = run([in_bin('wasm-dis'), wasm] + FEATURE_OPTS)
+        all_funcs = re.findall(r'\n [(]func [$](\S+)', wat)
```

Comment:
Interesting, I haven't seen square brackets used as a form of escape before. I think backslashes would work, too, and might be less surprising. It also might be good to stash this regex in compiled form in a global.

Reply:
Compiling the regex sgtm. About clarity, though, I feel that `[(]` is better than `\(` because the slash is similar to `\S` etc. where it is *not* escaping anything. The list of things that need escaping differs between regex impls AFAIK, but doing `[X]` always works.